test/controllers/update/update.manage.trusts.review.the.trust.controller.spec.ts

```diff
@@ -382,6 +407,127 @@ describe('Update - Manage Trusts - Review the trust', () => {
       expect(resp.text).toContain(errorMessage);
     });
 
+    // Test the trust ceased date validation combinations
+    test.each([
+      [
```

Comment:
This is repetition of the test cases defined for the test above. I'd ideally like to define them once and reference in both tests but I had no luck in doing so (trouble defining with the Jest types). @markpit any ideas?

Reply:
you could add the test.each scenarios to the test above, although the test above is testing the 'blue box' version and is checking the BO check boxes are not displayed.
if the test.each scenarios are the same then test.each() takes an array [], so that array could be defined as a const at top of test file and used something like test.each(myTestScenarios)

Reply:
Have implemented as you suggested and no issues with Jest types this time.